docs/cn/MatrixOne/Get-Started/0.6-basic-sql.md

```diff
@@ -0,0 +1,225 @@
+USE dbdemo;
+```
+
+- 使用 `SHOW TABLES` 语句查看数据库中所有的表，例如：
+
+```sql
+SHOW TABLES FROM dbdemo;
+```
+
+- 删除数据库，使用`DROP DATABASE` 语句：
+
+```sql
+DROP DATABASE dbdemo;
+```
+
+## 创建、查看和删除表
+
+- 创建表，使用 `CREATE TABLE` 语句：
+
+```sql
+CREATE TABLE table_name column_name data_type constraint;
+```
+
+例如，要创建一个名为 *person* 的表，包括编号、名字、生日等字段，可使用以下语句：
+
+```sql
+CREATE TABLE person (
+    id INT(11),
+    name VARCHAR(255),
+    birthday DATE
+    );
+```
+
+- 查看新建的表，使用 `SHOW CREATE` 语句：
+
+```sql
+SHOW CREATE table person;
+```
+
+结果如下所示：
+
+```sql
++--------+-----------------------------------------------------------------------------------------------------------------+
+| Table  | Create Table                                                                                                    |
++--------+-----------------------------------------------------------------------------------------------------------------+
+| person | CREATE TABLE `person` (
+`id` INT DEFAULT NULL,
+`name` VARCHAR(255) DEFAULT NULL,
+`birthday` DATE DEFAULT NULL
+) |
++--------+-----------------------------------------------------------------------------------------------------------------+
+1 row in set (0.01 sec)
+```
+
+- 删除表，使用 `DROP TABLE` 语句：
+
+```sql
+DROP TABLE person;
+```
+
+## 增价、删出、修改表记录
+
+通用的 DML 即增删改表记录，基本的语句为 `INSERT`，`UPDATE` 和 `DELETE`。
+
+- 向表中插入数据，使用 `INSERT` 语句：
+
+```sql
+INSERT INTO person VALUES(1,'tom','20170912');
+```
+
+- 向表内插入包含部分字段数据的表记录，使用 `INSERT` 语句，例如：
+
+```sql
+INSERT INTO person(id,name) VALUES('2','bob');
+```
+
+- 向表内修改表记录的部分字段数据，使用 `UPDATE` 语句，例如：
+
+```sql
+UPDATE person SET birthday='20180808' WHERE id=2;
+```
+
+- 向表内删除部分表记录，使用 `DELETE` 语句，例如：
+
+```sql
+DELETE FROM person WHERE id=2;
+```
+
+!!!note
+   `UPDATE` 和 `DELETE` 操作如果不带 `WHERE` 过滤条件则是对全表进行操作。
+
+## 查询数据
+
+DQL用于从一个表或多个表检索所需的数据行。
+
+- 查看表内数据，使用 `SELECT` 语句：
+
+```sql
+SELECT * FROM person;
+```
+
+结果如下所示：
+
+```sql
++------+------+------------+
+| id   | name | birthday   |
++------+------+------------+
+|    1 | tom  | 2017-09-12 |
++------+------+------------+
+1 row in set (0.00 sec)
+```
+
+- 查询某一列，使用 `SELECT` 语句加上要查询的列名：
+
+```sql
+SELECT name FROM person;
++------+
+| name |
++------+
+| tom  |
++------+
+1 rows in set (0.00 sec)
+```
+
+使用 `WHERE` 子句，对所有记录进行是否符合条件的筛选后再返回。例如：
+
+```sql
+SELECT * FROM person where id<5;
+```
+
+结果如下所示：
+
+```sql
++------+------+------------+
+| id   | name | birthday   |
++------+------+------------+
+|    1 | tom  | 2017-09-12 |
++------+------+------------+
+1 row in set (0.00 sec)
+```
+
+## 创建、授权和删除用户
+
+- 使用 `CREATE USER` 语句创建一个用户 *mouser*，密码为 *111*：
+
+```sql
+CREATE USER mouser IDENTIFIED BY '111';
+```
+
+- 创建角色 *role_r1*、*role_r2*、*role_r3*：
+
+```sql
+CREATE ROLE role_r1,role_r2,role_r3;
+```
+
+- 将角色 *role_r1* 赋予用户 *mouser*：
+
+```sql
+GRANT role_r1 to mouser;
+```
+
+- 授权角色 *role_r1* 可以在 *dbdemo* 数据库里建表的权限：
+
+```sql
+GRANT create table on database * to role_r1;
+```
+
+- 查询用户 *mouser* 的权限：
+
+```sql
+SHOW GRANTS for mouser;
```

Comment:
not sure if this is a good example. show grants should give grants list instead of nothing. @domingozhang please help verify.

Reply:
SHOW GRANTS  --> show grants for root